styles.css

```diff
@@ -17,278 +17,7 @@ body {
   color: white;
 }
 
-
```

Comment:
As long as you can, try not to have code that is unused, it's a really bad practice because you'll end up with a ton of shit in your project which only makes it more complex, and the bigger the project, the most difficult it's gonna be to find out if you are using that code or not.

Reply:
This is more difficult when using third-party libraries, since you might only want to use certain part of the library but that might often require you to add the entire library to your project